data/drak/drak.txt

```diff
@@ -234,6 +234,8 @@ ship "Archon"
 	turret 55 16 "Drak Distancer"
 	explode "nuke explosion" 3
 	"final explode" "final explosion large"
+	description "Very little is known about Archons other than that they significantly exceed the spacefaring prowess of any known species. From external observation, they appear to be a mix of organic and cybernetic elements. They are known to guard noteworthy locations and sapient species - or the remains thereof."
+	description "	It is unknown whether Archons are grown, created, or raised. Likewise, while they are identified as 'Drak' by most ships' identification systems, it is unknown whether or not they are Drak, servants or agents of the Drak, or another thing entirely."
```

Comment:
I'd go with something much simpler. The description should not simply be a list of those ideas that we've seen scattered throughout the story of the game. (Especially because should this ever become readable in game, we shouldn't basically be spoiling ideas that are mentioned by the story.) See #3070 for the most likely way in which we'd be able to see the description of any ship in game, so the description should fit the setting there (i.e. you're flying around in space with the ship in your system).
```suggestion
	description `The Archon's presence is chilling. The "ship" has an almost organic appearance, but your ship's scanners give you no discernible information on its construction. What exactly it is you do not know, but you can feel that it is powerful.`
```